elastic/kibana
# [Console] Telemetry (part 1)

## Summary

Adds telemetry tracking for app start up and tracking user issued requests.

### How to review
- Use of `ui_metrics` should be updated after https://github.com/elastic/kibana/pull/51972 is merged (current use is from legacy)
- Check that data is sufficiently anonymised.
- ~Currently, ui metric counts look like: `{ 'GET _search|{indices}/_search': 16 }`~ ui metric count events look like `POST sql.query` as they appear in the pre-determined JSON files used for completions (e.g., `x-pack/legacy/plugins/console_extensions/spec/generated/sql.query.json`) . this is to preserve anonymity while also providing us with information to make decisions like: should we build a UI for this? Will this give us enough information to make a decision like this or do we need more information about the sequence in which requests are run?
